simulation/constants.py

```diff
@@ -0,0 +1,56 @@
+import enum
+from typing import Dict
+
+
+class PropertyType(enum.Enum):
+    HOUSE = 0
+    FLAT = 1
+    BUNGALOW = 2
+
+
+class BuiltForm(enum.Enum):
+    MID_TERRACE = 0
+    SEMI_DETACHED = 1
+    DETACHED = 2
+    END_TERRACE = 3
+
+
+class OccupantType(enum.Enum):
+    OWNER_OCCUPIER = 0
+    RENTER_PRIVATE = 1
+    RENTER_SOCIAL = 1
+
+
+class Epc(enum.Enum):
+    A = 0
+    B = 1
+    C = 2
+    D = 3
+    E = 4
+    F = 5
+    G = 6
+
+
+class HeatingSystem(enum.Enum):
+    BOILER_GAS = 0
+    BOILER_OIL = 1
+    BOILER_ELECTRIC = 2
+    HEAT_PUMP_AIR_SOURCE = 3
+    HEAT_PUMP_GROUND_SOURCE = 4
```

Comment:
Note we could play the same game as above e.g 
```suggestion
    BOILER_GAS = 0
    BOILER_OIL = 0
    BOILER_ELECTRIC = 0
    HEAT_PUMP_AIR_SOURCE = 1
    HEAT_PUMP_GROUND_SOURCE = 1
```
so that when we apply `HeatingSystem.value` it turns into `is_heat_pump`. Or is this an abuse of `enum.Enum` and really they should all be different? @tomwphillips ?

Reply:
Like I said above it's a bit unexpected IMO.